src/components/SignupConfirmation.tsx

```diff
@@ -0,0 +1,161 @@
+import React, {useState} from 'react';
```

Comment:
I would prefer if this was in a folder: `src/components/signup-confirmation/SignupConfirmation.tsx`